ete3/tools/ete_evol.py

```diff
@@ -489,13 +490,13 @@ def clean_exit(a, b):
     os.chdir(fullpath)
 
     proc = Popen("%s tmp.ctl" %binary, stdout=PIPE, stdin=PIPE, shell=True)
-    proc.stdin.write('\n') # in case codeml/slr asks something
+    proc.stdin.write(b'\n') # in case codeml/slr asks something
     job, err = proc.communicate()
     if err is not None or b'error' in job or b'Error' in job:
-        print("ERROR: inside CodeML!!\n" + job)
+        print((b"ERROR: inside CodeML!!\n" + job).decode())
         return (None, None)
     os.chdir(hlddir)
-    return os.path.join(fullpath,'out'), model_obj.name
+    return os.path.join(fullpath, 'out').encode(), model_obj.name.encode()
```

Comment:
I take it this function needs to return bytes?
The encode/decode dance is this code is brittle.
Not sure this will work if the user's environment isn't UTF8 compatible.

I also didn't check but, are tests covering this code?
Wondering how this behaves on py2 vs py3.